config/covid-19/forms/app/covid19_rdt_provision.properties.json

```diff
@@ -0,0 +1,9 @@
+{
+  "icon": "icon-follow-up",
+  "title": "RDT Covid-19 - Provision",
+  "context": {
+    "person": true,
+    "place": false,
+    "expression": "contact.type === 'person'"
```

Comment:
I don't think it's currently possible but the expression being able to detect if we are on an actual device would be cool. As you cannot complete this form without the app we should just not show it if they're not on an android device.

Reply:
While I agree this would be a good feature to have in `expression`, it is handy to be able to view and edit already submitted forms on a desktop. This is allowed b/c the required RDT fields are populated when it was created on Android w/ RDT app.  I think this is enough of a real world to keep as is! (even though we can't actually change it to require a device ; )

Reply:
> it is handy to be able to view and edit already submitted forms on a desktop. This is allowed b/c the required RDT fields are populated when it was created on Android w/ RDT app. 


Viewing and editing is not controlled by this expression. Only whether it displays in the drop up. 

I do agree that would be a potential feature for another time.

Reply:
TIL - thanks!!